src/vehicle.cpp

```diff
@@ -3427,6 +3457,12 @@ int vehicle::safe_ground_velocity( const bool fueled ) const
     return mps_to_vmiph( safe_in_mps );
 }
 
+int vehicle::safe_rotor_velocity( const bool fueled ) const
+{
+    double max_air_mps = std::sqrt( lift_thrust_of_rotorcraft( fueled ) / coeff_air_drag() );
```

Comment:
```suggestion
    double max_air_mps = std::sqrt( lift_thrust_of_rotorcraft( fueled, true ) / coeff_air_drag() );
```
you should calculate safe rotor velocity from the safe lift_thrust_of_rotorcraft